fdbserver/workloads/EncryptionOps.actor.cpp

```diff
@@ -127,41 +126,38 @@ struct EncryptionOpsWorkload : TestWorkload {
 		throw internal_error();
 	}
 
-	void updateEncryptionKey(StreamCipherKey* cipherKey) {
-		auto start = std::chrono::high_resolution_clock::now();
-		applyHmacKeyDerivationFunc(cipherKey, hmacGenerator.get(), arena);
-		auto end = std::chrono::high_resolution_clock::now();
-
-		metrics->updateKeyDerivationTime(std::chrono::duration<double, std::nano>(end - start).count());
-	}
-
-	StringRef doEncryption(const StreamCipherKey* key, uint8_t* payload, int len) {
-		EncryptionStreamCipher encryptor(key, iv);
+	StringRef doEncryption(Reference<BlobCipherKey> key, uint8_t* payload, int len, BlobCipherEncryptHeader* header) {
+		EncryptBlobCipherAes265Ctr encryptor(key, iv);
 
 		auto start = std::chrono::high_resolution_clock::now();
-		auto encrypted = encryptor.encrypt(buff.get(), len, arena);
-		encryptor.finish(arena);
+		auto encrypted = encryptor.encrypt(buff.get(), len, header, arena);
 		auto end = std::chrono::high_resolution_clock::now();
```

Comment:
I don't know which is "best practice" or not but we do also have timer_monotonic() in FDB? That might also save you some of the conversion math too

Reply:
Yeah I couldn't find any standard usage across the board, hence, went with `chrono`; however, looking at OffsetTimer (which is used for timer_monotonic) the resolution is in seconds, which would work fine for encryption usecase. Will update the usage.
Thanks for pointing out.

Reply:
Ahh. It took me some time to page all the context IN, `chrono` was used only in the test program given using simulation clock is not useful for benchmarking purposes, however, for production code `now()` is used for all time keeping needs, which seems appropriate to me.